src/main/java/io/reactivex/Flowable.java

```diff
@@ -15774,34 +15761,31 @@ public final Completable switchMapCompletableDelayError(@NonNull Function<? supe
     }
 
     /**
-     * Returns a Flowable that only emits those items emitted by the source Publisher that are not followed
-     * by another emitted item within a specified time window.
+     * Alias to {@link #debounce(long, TimeUnit)}
```

Comment:
Please keep the description as the first sentence and these as the second one or as a braced comment on the first sentence.

```
Returns a Flowable that only emits those items emitted by the source Publisher that are not followed
by another emitted item within a specified time window (alias to {@link #debounce(long, TimeUnit)}).
```